test/integration/transaction.test.js

```diff
@@ -58,8 +59,9 @@ describe('Transactions', function () {
 
   describe('CLS', function () {
     beforeEach(async function () {
-      this.namespace = cls.createNamespace('sequelize');
-      if (/^[45]/.test(Sequelize.version)) {
+
+      this.namespace = (/^[6]/.test(Sequelize.version) ? clsh : cls).createNamespace('sequelize');
```

Comment:
sequelize changed the namespace they use?

Reply:
yes they did.

Reply:
https://sequelize.org/master/manual/upgrade-to-v6.html